completion_aggregator/api/v1/views.py

```diff
@@ -258,10 +258,11 @@ class CompletionDetailView(CompletionViewMixin, APIView):
 
         username (optional):
             The username of the specified user for whom the course data is being
-            accessed.
+            accessed. If a non-staff users try to access another user's data they
```

Comment:
@Metfriet, nit: remove `a` since the subject is in plural.

Reply:
@lgp171188 fixed, sorry for the hassle